src/objects.js

```diff
@@ -1,41 +1,55 @@
 const createPerson = (name, age) => {
-  // your code here
+  const Person = {
+    name: name,
+    age: age,
+  }
+    return Person;
 };
 
 const getName = object => {
-  // your code here
+  return object.name;
 };
 
 const getProperty = (property, object) => {
-  // your code here
+  return object[property];
 };
 
 const hasProperty = (property, object) => {
-  // your code here
+  return object.hasOwnProperty(property);
 };
 
 const isOver65 = person => {
-  // your code here
+  if (person.age > 65) {
+    return true;
+  } else {
+    return false;
+  }
 };
 
 const getAges = people => {
-  // your code here
+  return people.map((ages)=> ages.age);
 };
 
 const findByName = (name, people) => {
-  // your code here
+  return people.find((item) => item.name === name)
 };
 
 const findHondas = cars => {
-  // your code here
+  return cars.filter(company => company.manufacturer === "Honda");
 };
 
 const averageAge = people => {
-  // your code here
+  return people.reduce((acc, curr) => acc + curr.age, 0) / people.length;
 };
 
 const createTalkingPerson = (name, age) => {
-  // your code here
+  return {
+    name: name,
+    age: age,
```

Comment:
Minor point, but there is a shorthand for this that shortens the code a little bit:
```suggestion
    name,
    age,
```